external/source/shellcode/osx/stager/main.c

```diff
@@ -66,12 +66,9 @@ int main(int argc, char** argv)
   }
 
   NSCreateObjectFileImageFromMemory_ptr NSCreateObjectFileImageFromMemory_func = (void*)find_symbol(dyld, "_NSCreateObjectFileImageFromMemory");
-  if (!NSCreateObjectFileImageFromMemory_func) {
+  while (!NSCreateObjectFileImageFromMemory_func) {
```

Comment:
Should the return values of `find_macho` or `find_symbol` be validated to prevent looping forever?

Reply:
Thanks @bcoles , this is fixed now, although I'm not sure in what scenario it would happen (maybe a future version of OSX?).